src/log_outgoing_requests/models.py

```diff
@@ -61,3 +76,8 @@ def __str__(self):
         return ("{hostname} at {date}").format(
             hostname=self.hostname, date=self.timestamp
         )
+
+    @cached_property
+    def query_params(self):
+        parsed_url = urlparse(self.url)
+        return parsed_url.query
```

Comment:
This is ok for now since we only take the query, but if we'd want more url components you'd cache the result and access that like so:

```python
@cached_property
def url_parsed(self):
    return urlparse(self.url)

@property
def query_params(self):
    return self.url_parsed.query

@property
def scheme(self):
    return self.url_parsed.scheme
```